app/views/hub/components/_main_menu.html.erb

```diff
@@ -10,7 +10,7 @@
           <div class="logo-text">The Hub</div>
         </div>
 
-        <% if current_user %>
+        <% if current_user&.forced_password_reset_at? || current_user&.admin? %>
```

Comment:
We now have code that redirects the user to the password change page when needed, so there's no need for this.